Jenkinsfile

```diff
@@ -1 +1 @@
-buildPlugin()
+buildPlugin(jenkinsVersions: [null, '2.102'])
```

Comment:
Pending https://github.com/jenkinsci/workflow-multibranch-plugin/pull/69, this will quietly be ignored until merged. I did the equivalent build (on Linux) locally.